tests/config/test_themes.py

```diff
@@ -36,41 +35,14 @@ def test_all_themes():
 )
 def test_builtin_theme_completeness(theme_name):
     theme = THEMES[theme_name]
-    styles_in_theme = {style[0] for style in theme}
+    theme_styles = theme.STYLES
+    theme_colors = theme.Color
```

Comment:
If I move this before generating all the tests then I would need to add and update the NEW_THEMES dict at each commit, it would be cleaner to create it only once. I could move it to the commit where new themes were compared with old ones, i.e., the previous commit.

Reply:
I was meaning that you define the new themes, then add the test for completeness, then generate + test, then compare new vs old with the test.

In any case I'd suggest splitting the new theme completeness and removal tests into separate commits.